src/views/CharacterBuild.js

```diff
@@ -22,12 +46,41 @@ export default function CharacterBuild() {
     <>
       {map(keys(charBuild), (stats, idx) => {
         return (
-          <Box key={idx} style={{ display: "flex" }}>
-            <Box style={{ justifyContent: "flex-start", minWidth: "250px" }}>
+          <Box
+            key={idx}
+            style={{ display: "flex", alignItems: "center" }}
+            marginBottom="20px"
+          >
+            <Box
+              style={{
+                justifyContent: "flex-start",
+                minWidth: "250px",
+                textAlign: "right"
+              }}
+              marginRight="20px"
+            >
               {stats}
             </Box>
             <Box style={{ justifyContent: "flex-end", minWidth: "100px" }}>
-              {get(charBuild, stats)}
+              <TextField
+                type={stats === "character_name" ? "text" : "number"}
+                label={stats}
+                variant="outlined"
```

Comment:
Remove variant. The default with just line looks cleaner

Reply:
the textbox is like invisible without an outline, unless u want it to be filled
or u can change the background color

Reply:
thats fine, just change the on hover color to white or smth
![image](https://user-images.githubusercontent.com/51217000/173734164-f6127c80-a512-4f56-996e-5999a1eff90f.png)